tests/status_response/test_java.py

```diff
@@ -100,5 +102,696 @@ class TestJavaStatusVersion(BaseStatusResponseTest):
+                            "ad8ce1aa99d0b8e48b9ae3a6a5e0a08be28083c680e5a19ee3a4b2e6b5afe5b392e396a5cc8be281b7e49986e4b18be19"
+                            "c98d0b1e48b9ae3a6a5e0a09bcba326c680e392b8e781a7e5b38ae68095c5b1e18ba3e28186e5a682e3a6b0e689b4e5b3"
+                            "8ae2868de380abcba3e49786e4878ce390b1e6b9a1e4ab9ce196b1e786a0e68da2e29885e48280e3a084e6b1afe5abb2e"
+                            "4a6bde48e83e182b6e29786e0b88ce1a497e380ade6a19ce18390e0ada8e69a96e280ad0cd480e795a1e5bba8e19789e6"
+                            "8cbbe29a96e2838ce4ae8ce19a9be38cb5e1b084e5878ce18dbbe79897e0b2ace5b299e3aeb0e789a5e0b3a6e48384e18"
+                            "5b1e18ba3e28686e5a19be39cb4e68d9fe48b90e3a6b9e68cabe1809606cc80e398b3e7a1b5e4ab9ce5b791e18dbbe39a"
+                            "b7e683aee0ae8de19c98e2b8b770e3a0a8e7acabe39ba6e2b98ce1a698e1a4bae38883e6919ce28088e78ca8e29996e2b"
+                            "1aee1ac99ceb6e2b8b0e1b1a0e3a4bde3a8aae59ba0e6ba8ce1af9de396b9e38085e6819ce482b8e18081e18290e0b88c"
+                            "e5ac9ce3a6b2e6a5abe1a39ce3a388e786a1e38c82e6b6a5e0b198e1a297d0b8e78ba6e0b6b9e0a08be18093e485a0e1a"
+                            "599e3a4b9e791a9e49b8ae4a6bde18cabe583a0e0b7ade1b599e382b6e789b2e6ab9ee19791e1ae93e18287e29787e4ae"
+                            "8ce19a98e38cb9e5a898e29685e7adb3d8b5e4b0ade5ae9be398b2e3888100e4a190e38cabe69a96e0b2ade1b399e39eb"
+                            "ae685b2e4ab8ee18685e7aca3e39ba6c2aee0ae8ce19c9c32e5b894e19781e68db3e19bb6e2b28ce1b299e1a283e2b8b2"
+                            "e5b1a0e0a384e28188e59a87e4aface5a19de398bae185b4e5b1a2e68384e185b1e28b93d786e4ae8ce1a098e3a0aee78"
+                            "9ace3a09ce28cabe79db7e6b98dc69ae19c98e398b2e6819cd080e6a2a8e49bb6e4b2ace5ae9ce392bae698bae6ab98e3"
+                            "a7a1e28cabe79db7e6b98de5b39ae1ae81e39cb000"
+                        )
+                    ).decode("utf-8"),
+                    "fmlNetworkVersion": 3,
+                    "mods": [],
+                    "truncated": True,
+                }
+            )
+        )
+        assert value is not None
+        return value
+
+
+@BaseStatusResponseTest.construct
+class TestForgeDataV1(TestJavaStatusResponse):
+    RAW = {
+        "description": {"text": "A Minecraft Server"},
+        "players": {"max": 20, "online": 0},
+        "version": {"name": "1.12.2", "protocol": 340},
+        "modinfo": {
+            "type": "FML",
+            "modList": [
+                {"modid": "minecraft", "version": "1.12.2"},
+                {"modid": "mcp", "version": "9.42"},
+                {"modid": "FML", "version": "8.0.99.99"},
+                {"modid": "forge", "version": "14.23.5.2859"},
+            ],
+        },
+    }
+
+    EXPECTED_VALUES = [
+        ("players", JavaStatusPlayers(0, 20, None)),
+        ("version", JavaStatusVersion("1.12.2", 340)),
+        ("motd", Motd.parse("A Minecraft Server", bedrock=False)),
```

Comment:
Tests fail because of my mistake, sorry, it should be:

```suggestion
        ("motd", Motd.parse({"text": "A Minecraft Server"}, bedrock=False)),
```

Reply:
Actually, never mind if we will delete this part anyway.